helpers/lib/fonts.js

```diff
@@ -84,58 +82,90 @@ const fontProviders = {
+                if (!key.endsWith(fontKeySuffix)) {
+                    return collectedFonts;
+                }
 
-        const splits = value.split('_');
-        const provider = splits[0];
+                const splits = value.split('_');
+                const provider = splits[0];
 
-        if (typeof fontProviders[provider] === 'undefined') {
-            return;
-        }
+                if (typeof fontProviders[provider] === 'undefined') {
+                    return collectedFonts;
+                }
+
+                if (typeof collectedFonts[provider] === 'undefined') {
+                    collectedFonts[provider] = [];
+                }
 
-        if (typeof collectedFonts[provider] === 'undefined') {
-            collectedFonts[provider] = [];
-        }
+                collectedFonts[provider].push(value);
 
-        collectedFonts[provider].push(value);
-    });
+                return collectedFonts;
+            },
+            {}
+        );
 
     // Parse font strings based on provider
-    const parsedFonts = _.mapValues(collectedFonts, function(value, key) {
-        return fontProviders[key].parser(value);
-    });
+    const parsedFonts = Object.entries(collectedFonts).reduce(
+        (
+            parsedFonts,
+            [key, value]
+        ) => {
+            parsedFonts[key] = fontProviders[key].parser(value);
+
+            return parsedFonts;
+        },
+        {}
+    );
 
     // Format output based on requested format
     switch(format) {
     case 'linkElements':
-        
-        const formattedFonts = _.mapValues(parsedFonts, function(value, key) {
-            return fontProviders[key].buildLink(value, options.fontDisplay);
-        });
-        return new handlebars.SafeString(_.values(formattedFonts).join(''));
+        const formattedFonts = Object.entries(parsedFonts).reduce(
+            (
+                formattedFonts,
+                [key, value]
+            ) => {
+                formattedFonts[key] = fontProviders[key].buildLink(value, options.fontDisplay);
+
+                return formattedFonts;
+            },
+            {}
+        );
+
+        return new handlebars.SafeString(Object.entries(formattedFonts).map(([key, value]) => value).join(''));
 
     case 'webFontLoaderConfig':
         // Build configs
-        const configs = _.mapValues(parsedFonts, function(value, key) {
-            return fontProviders[key].buildFontLoaderConfig(value);
-        });
-
-        // Merge them
-        const fontLoaderConfig = {};
-        _.each(configs, function(config) {
-            return Object.assign(fontLoaderConfig, config);
-        });
-        return fontLoaderConfig;
+        return Object.entries(parsedFonts).reduce(
+            (
+                configs,
+                [key, value]
+            ) => {
+                configs[key] = fontProviders[key].buildFontLoaderConfig(value);
+
+                return Object.entries(configs).reduce(
```

Comment:
Can't be it simplified to 
`
return {...configs, ...fontProviders[key].buildFontLoaderConfig(value)};
`?

Reply:
Eslint complain on this sadly. Worth investigating the reason in future PR definitely.

```
/Users/patrick.edelman/dev/paper-handlebars/helpers/lib/fonts.js
  154:31  error  Parsing error: Unexpected token ..
```

Reply:
@pedelman, @jairo-bc the reason is that we have specified to support ES6 only in our .eslintrc, just need to update the config to support the newer JS syntax. I had a similar issue with Stencil-CLI previously.
Created a PR with the updated ESLint: https://github.com/bigcommerce/paper-handlebars/pull/103